src/styles/common/c-btn.scss

```diff
@@ -11,8 +11,8 @@
 
```

Comment:
Why are we using two different classes for buttons? We already have a `c-button` component, I don't know when this file came up, but I'm pretty sure that we don't need it.